lib/sms/sms.c

```diff
@@ -163,7 +163,7 @@ static void sms_at_cmd_handler_cmt(const char *at_notif)
 
 	__ASSERT_NO_MSG(at_notif != NULL);
```

Comment:
Could you remove `__ASSERT_NO_MSG` and change to `if`? So there'll  be no "crash" of app.
After that send ACK even before this check.